SQF/dayz_code/stringtable.xml

```diff
@@ -1376,23 +1376,23 @@
 			<Spanish>Se las arregló para conseguirse un conjunto de trapos se desgarre la ropa.</Spanish>
 			<Czech>Z roztrhaného oblečení jste získali jednu hromádku hadrů.</Czech>
 			<French>Vous avez réussi à faire un chiffon de ces vêtements.</French>
-			<German>Du schaffst es, einen kleinen Haufen Lumpen zurecht zu reißen.</German>
+			<German>Du schaffst es einen kleinen Haufen Lumpen zurechtzureißen.</German>
```

Comment:
I recommend to let the comma. It is not a must have but for a better sentence structure you should let it. The comma introduces the infinitive with "zu" so we can use it there.